### DISCUSSION TITLE
Reality 项目下同一个文件夹中两个package 的问题

### DISCUSSION THREAD

**Discussion by nathnial** [XTLS/Xray-core#2081] [Q&A]
reality 是怎么做到，同一个文件夹下有两个package 的。main & reality

**Comment by RPRX** [comment]
因为有一个没用的文件 https://github.com/XTLS/REALITY/blob/main/generate_cert.go

**Reply by nathnial** [reply]
那就是说咱们当前版本没有使用自己生成的cert.pem 证书吗。这个generrate_cert.go 我正好看过了，是生成cert.pem证书, & key.pem的。
